### PR TITLE
Add full support for fullobjects support for AT content types

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Comment by tisto** [comment]
@sneridagh test failures on py3:

```
Error in test test_get_content_returns_fullobjects (plone.restapi.tests.test_content_get.TestContentATGet)
Traceback (most recent call last):
  File "/opt/python/3.6.7/lib/python3.6/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/opt/python/3.6.7/lib/python3.6/unittest/case.py", line 601, in run
    self.setUp()
  File "/home/travis/build/plone/plone.restapi/src/plone/restapi/tests/test_content_get.py", line 174, in setUp
    self.portal = self.layer['portal']
  File "/home/travis/buildout-cache/eggs/plone.testing-7.0.0-py3.6.egg/plone/testing/layer.py", line 30, in __getitem__
    raise KeyError(key)
KeyError: 'portal'
```

**Comment by davisagli** [comment]
You'll have to skip Archetypes tests when Archetypes is not present (i.e. on Python 3). Here's what we did for other tests:

    from plone.restapi.testing import HAS_AT
    import unittest

    # at the top of setUp():
    if not HAS_AT:
        raise unittest.SkipTest('Skip tests if Archetypes is not present')

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

File: src/plone/restapi/testing.py [hunk 1]
```diff
@@ -284,6 +284,8 @@ def setUpZope(self, app, configurationContext):
             z2.installProduct(app, 'plone.restapi')
 
         def setUpPloneSite(self, portal):
+            portal.acl_users.userFolderAddUser(
```
[the remark this answers is not in the document]
Comment (sneridagh) [reply src/plone/restapi/testing.py]: No, it's not. The AT fixture didn't had any user set, we need it to behave the same way than the DX one:
https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/testing.py#L229

**Comment by sneridagh** [comment]
@tisto I forgot about this one!

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**